asserter/asserter.go

```diff
@@ -261,3 +305,20 @@ func (a *Asserter) OperationSuccessful(operation *types.Operation) (bool, error)
 
 	return val, nil
 }
+
+func getValidationConfig(validationFilePath string) (*Validations, error) {
+	validationConfig := &Validations{
+		Enabled: false,
```

Comment:
should we enable it by default?

Reply:
I think to support backward compatibility, it makes sense to disable it by default.